src/api/routes/address.ts

```diff
@@ -230,5 +237,32 @@ export function createAddressRouter(db: DataStore, chainId: ChainID): RouterWith
     }
   });
 
+  router.getAsync('/:stx_address/nft_events', async (req, res) => {
+    // get recent asset event associated with address
+    const stxAddress = req.params['stx_address'];
+    if (!isValidPrincipal(stxAddress)) {
+      return res.status(400).json({ error: `invalid STX address "${stxAddress}"` });
+    }
+
+    const limit = parseAssetsQueryLimit(req.query.limit ?? 20);
+    const offset = parsePagingQueryInput(req.query.offset ?? 0);
+    const response = await db.getAddressNFTEvent({
+      stxAddress,
+      limit,
+      offset,
+    });
+    const nft_events = response.results.map(row => ({
+      ...row,
```

Comment:
Nit: can this object be explicitly defined rather than the spread operator? If extra properties end up in that row object in the future, they can unintentionally end up being sent to the client.